### PR TITLE
File Added : Uploaded file changes 

### PR DESCRIPTION
**Pull request by SnehalAhir**

### CODE REVIEW

**Review by SamikshaWani — APPROVED** [review]

**Review by arshad05786 — APPROVED** [review]

File: Snehal/HTML/Assignment 5/aboutself.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+          <meta name="viewport" content="width=device-width, initial-scale=1.0">
+          <title>Website Snehal</title>
+          <link rel="stylesheet" href="effect.css">
+    </head>
+    <body>
+        <div class="divmain">
+        <header>
+            <h1>Snehal Ahir</h1>
+        </header>
+            <i><h1 class="heading1">Aboutme</h1></i>
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment 5/aboutself.html:15]: check indentation

File: Snehal/HTML/Assignment 5/aboutself.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            <p class="p1">
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment 5/aboutself.html:16]: check indentation

File: Snehal/HTML/Assignment 5/aboutself.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+                    </li>
+                    <li>
+                        <a href="gallary.html" target="blank">My Gallary</a>
+                    </li>
+                </ul>
+            </nav>
+        </section>
+        </div>
+    </body>
+</html>
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment 5/aboutself.html:41]: add extra line space after </html>

File: Snehal/HTML/Assignment 5/effect.css [hunk 4]
```diff
@@ -0,0 +1,36 @@
+header{
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment 5/effect.css:1]: space required before curly bracket

File: Snehal/HTML/Assignment 5/effect.css [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    height:100%; width:100%;
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment 5/effect.css:2]: space required after :

File: Snehal/HTML/Assignment 5/effect.css [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+    text-align: left;
+    border: 1px solid black;
+    border-color: white;
+}
+nav{
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment 5/effect.css:7]: space required before curly bracket

File: Snehal/HTML/Assignment 5/effect.css [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+    float: center;
+    width: 99.7%;
+    height: 200px;
+}
+nav ul{
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment 5/effect.css:12]: space required before curly bracket

File: Snehal/HTML/Assignment 5/effect.css [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+    float: center;
+    list-style: square;
+    text-align: justify;
+}
+.divmain{
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment 5/effect.css:17]: space required before curly bracket

File: Snehal/HTML/Assignment 5/effect.css [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+    float: center;
+    text-align: center;
+    border: 2px solid black;
+    background-image: url(bird.jpeg);
+    width: 100%;
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+.pl{
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment 5/effect.css:26]: space required before curly bracket

File: Snehal/HTML/Assignment 5/effect.css [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+    text-align: left;
+    border: 1px solid black;
+    height: 100%;
+    width: 99.5%;
+}
+.div2{
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment 5/effect.css:32]: space required before curly bracket

File: Snehal/HTML/Assignment 5/effect.css [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+    height: 120px;
+    width: 100%;
+    vertical-align:bottom;
+}
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment 5/effect.css:36]: add one line space at the end of file

File: Snehal/HTML/Assignment3/assinment_4.html [hunk 12]
```diff
@@ -3,24 +3,21 @@
...
+            <img src="index.jpeg" width="400" height="300" />
+            <p style="width: 900px; margin-left: 10px">
+                Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.vContact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.
             </p>
         </div>
     </body>
-</html> |
\ No newline at end of file
+</html>
+|
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment3/assinment_4.html]: only one line space required at the end of file

File: Snehal/HTML/Assignment4/Third.html [hunk 13]
```diff
@@ -1,87 +1,88 @@
...
-            <td>XSL Formatting Objects</td>
-        </tr>
-    </table>
-    
-</body>
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Document</title>
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment4/Third.html]: change title, title should be related to program

**Review by Pawan1005 — APPROVED** [review]

File: Snehal/HTML/Assignment4/first.html [hunk 14]
```diff
@@ -1,43 +1,44 @@
...
-    </style>
-</head>
-<body style="margin: 30px 300px;">
-        <table style="width: 50%;">
-
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Assignment1</title>
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment4/first.html]: change title in every file

File: Snehal/HTML/Assignment5/aboutself.html [hunk 15]
```diff
@@ -0,0 +1,41 @@
...
+          <meta name="viewport" content="width=device-width, initial-scale=1.0">
+          <title>Website Snehal</title>
+          <link rel="stylesheet" href="effect.css">
+    </head>
+    <body>
+        <div class="divmain">
+        <header>
+            <h1>Snehal Ahir</h1>
+        </header>
+            <i><h1 class="heading1">Aboutme</h1></i>
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment5/aboutself.html]: check indentation

File: Snehal/HTML/Assignment5/aboutself.html [hunk 16]
```diff
@@ -0,0 +1,41 @@
...
+            <p class="p1">
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment5/aboutself.html]: check indentation

File: Snehal/HTML/Assignment5/aboutself.html [hunk 17]
```diff
@@ -0,0 +1,41 @@
...
+                    </li>
+                    <li>
+                        <a href="gallary.html" target="blank">My Gallary</a>
+                    </li>
+                </ul>
+            </nav>
+        </section>
+        </div>
+    </body>
+</html>
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment5/aboutself.html]: add one line space at the end of file

File: Snehal/HTML/Assignment5/effect.css [hunk 18]
```diff
@@ -0,0 +1,36 @@
+header{
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment5/effect.css]: space required before curly bracket

File: Snehal/HTML/Assignment5/effect.css [hunk 19]
```diff
@@ -0,0 +1,36 @@
...
+    height:100%; width:100%;
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment5/effect.css]: space required after :

File: Snehal/HTML/Assignment5/effect.css [hunk 20]
```diff
@@ -0,0 +1,36 @@
...
+    text-align: left;
+    border: 1px solid black;
+    border-color: white;
+}
+nav{
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment5/effect.css]: space required before curly bracket

File: Snehal/HTML/Assignment5/effect.css [hunk 21]
```diff
@@ -0,0 +1,36 @@
...
+    float: center;
+    width: 99.7%;
+    height: 200px;
+}
+nav ul{
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment5/effect.css]: space required before curly bracket

File: Snehal/HTML/Assignment5/effect.css [hunk 22]
```diff
@@ -0,0 +1,36 @@
...
+    float: center;
+    list-style: square;
+    text-align: justify;
+}
+.divmain{
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment5/effect.css]: space required before curly bracket

File: Snehal/HTML/Assignment5/effect.css [hunk 23]
```diff
@@ -0,0 +1,36 @@
...
+    float: center;
+    text-align: center;
+    border: 2px solid black;
+    background-image: url(bird.jpeg);
+    width: 100%;
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+.pl{
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment5/effect.css]: space required before curly bracket

File: Snehal/HTML/Assignment5/effect.css [hunk 24]
```diff
@@ -0,0 +1,36 @@
...
+    text-align: left;
+    border: 1px solid black;
+    height: 100%;
+    width: 99.5%;
+}
+.div2{
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment5/effect.css]: space required before curly bracket

File: Snehal/HTML/Assignment5/effect.css [hunk 25]
```diff
@@ -0,0 +1,36 @@
...
+    height: 120px;
+    width: 100%;
+    vertical-align:bottom;
+}
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment5/effect.css]: one line space required at the end of file

File: Snehal/HTML/Assignment5/effect.css [hunk 26]
```diff
@@ -0,0 +1,36 @@
...
+.pl{
+    text-align: left;
+    border: 1px solid black;
+    height: 100%;
+    width: 99.5%;
+}
+.div2{
+    height: 120px;
+    width: 100%;
+    vertical-align:bottom;
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment5/effect.css]: space required after :

**Review by KomalShinde813 — CHANGES_REQUESTED** [review]

File: Snehal/HTML/Assignment4/Second.html [hunk 27]
```diff
@@ -4,11 +4,9 @@
...
-        <title>Assignment2</title>
+        <title>Jan's Spring Term</title>
         <style>
-            table,
-            th,
-            td {
+            table, th, td {
                 border-left: 12px solid grey;
                 border-right: 12px solid grey;
                 border-top: 2px solid grey;
```
Comment (SamikshaWani) [inline Snehal/HTML/Assignment4/Second.html:12]: give proper name to class not a ,b,c at line no 17,21,24

**Pull request closed without merging**